packages/wonder-blocks-timing/util/types.js

```diff
@@ -129,3 +129,102 @@ export interface IAnimationFrame {
      */
     clear(resolve?: boolean): void;
 }
+
+/**
```

Comment:
Maybe this comment would be useful in that file?

Reply:
Not sure. The interface is public API facing, but the implementation is not.